joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/ast/JumpStatementsTests.scala

```diff
@@ -0,0 +1,98 @@
+package io.joern.csharpsrc2cpg.querying.ast
+
+import io.joern.csharpsrc2cpg.testfixtures.CSharpCode2CpgFixture
+import io.shiftleft.codepropertygraph.generated.ControlStructureTypes
+import io.shiftleft.semanticcpg.language.*
+
+class JumpStatementsTests extends CSharpCode2CpgFixture {
+  "AST creation for jump statements" should {
+    "be correct for break" in {
+      val cpg = code(basicBoilerplate("""
+          |for (int i = 0; i < 10; i++) {
+          | break;
+          |}
+          |""".stripMargin))
+
+      inside(cpg.method("Main").controlStructure.controlStructureTypeExact(ControlStructureTypes.BREAK).l) {
```

Comment:
@karan-batavia 
I would recommend not to use `inside` as you cannot debug from the IDE code inside the `inside` block. Whereas if you take the return list and extract the element out of it like in the below example that I tried to rewrite part of this test case.

 In fact, it makes the code more readable.

e.g.

```
val List(breakStatement) = cpg.method("Main").controlStructure.controlStructureTypeExact(ControlStructureTypes.BREAK).l
breakStatement.controlStructureType shouldBe ControlStructureTypes.BREAK
breakStatement.code shouldBe "break;"
breakStatement.lineNumber shouldBe Some(10)
breakStatement.columnNumber shouldBe Some(1)
```

cc: @DavidBakerEffendi @tuxology

Reply:
@pandurangpatil we have been following this conventions for all the tests right from AST Creation.

Reply:
I've had success with the IDE, and the reason I want to use this is since it encourages using `failure` with helpful error messages which is readable from CLI failures.

Reply:
I noticed that but debugging the inside block is difficult. In go as well I was following in a similar fashion, in fact there would be some test cases following this style. However, I once got badly stuck while debugging the dataflow test. I spent almost half a day trying to understand why I was not able to monitor the variables inside the inside block. 

Then I removed the `inside` block and changed the code in the way shown in the example. I could debug it easily.

Going ahead you can follow the recommended structure.

Reply:
I did face some IDE issues. My middle ground for this is that for a breaking query (not just a failing test), I replicate the query above inside block and debug. Once its done, and test is fixed, I remove it and place it back in inside 😬